appcelerator/titanium_mobile
# [TIMOB-26560] Android: Removed deprecated Ti.UI.KEYBOARD_* constants

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26560

**Summary:**
- These keyboard type/appearance constants were deprecated in Titanium 6.0.0.
- You must now use the `Ti.UI.KEYBOARD_TYPE_*` constants instead.
- The API docs will be updated by PR #10465.

**Test:**
1. Build and run the below code on Android.
2. Verify that each keyboard constant is logged as "undefined".

```javascript
var keyboardTypeNames = [
	"Ti.UI.KEYBOARD_APPEARANCE_ALERT",
	"Ti.UI.KEYBOARD_ASCII",
	"Ti.UI.KEYBOARD_DECIMAL_PAD",
	"Ti.UI.KEYBOARD_DEFAULT",
	"Ti.UI.KEYBOARD_EMAIL",
	"Ti.UI.KEYBOARD_NAMEPHONE_PAD",
	"Ti.UI.KEYBOARD_NUMBERS_PUNCTUATION",
	"Ti.UI.KEYBOARD_NUMBER_PAD",
	"Ti.UI.KEYBOARD_PHONE_PAD",
	"Ti.UI.KEYBOARD_WEBSEARCH",
	"Ti.UI.KEYBOARD_TWITTER",
	"Ti.UI.KEYBOARD_URL",
];
for (var index = 0; index < keyboardTypeNames.length; index++) {
	var typeName = keyboardTypeNames[index];
	Ti.API.info(typeName + ": " + eval(typeName));
}
```
